packages/batch/tests/unit/SqsFifoPartialProcessor.test.ts

```diff
@@ -69,26 +57,22 @@ describe('Class: SqsFifoBatchProcessor', () => {
     });
   });
 
-  describe('Asynchronous SQS FIFO batch processing', () => {
-    test('SQS FIFO Batch processor with no failures', async () => {
+  describe('hronous SQS FIFO batch processing', () => {
```

Comment:
typo, but also:
We can probably remove this whole set of tests since we aren't doing the asynchronous processing anymore and this now just does the same thing as the first set of tests.

Reply:
Good point, removed the set.

See [here](https://github.com/aws-powertools/powertools-lambda-typescript/pull/1616#discussion_r1267773334) for why we don't have async behavior with SQSFifoHandler.